packages/fxa-content-server/app/tests/spec/views/mixins/third-party-auth-mixin.js

```diff
@@ -108,6 +109,15 @@ describe('views/mixins/third-party-auth-mixin', function () {
       assert.isFalse(view.logViewEvent.called);
     });
 
+    it('with third party auth set from other', () => {
+      Storage.factory('localStorage', windowMock).set(
+        'fxa_third_party_params',
+        'some params'
+      );
+      view.beforeRender();
+      assert.isFalse(view.completeSignIn.calledOnce);
```

Comment:
Q: what is this checking?
Wouldn't this pass if `view.completeSignIn` was called 0 times or 19 times?
Do we kno whow many times `view.completeSignIn` should be called and assert that it's been called exactly _n_ times?